docs/guides/guides/screenshots-and-videos.mdx

```diff
@@ -63,8 +63,8 @@ Videos are stored in the
 `cypress/videos` by default.
 
 After `cypress run` completes, Cypress compresses the video in order to save on
-file size. By default it compresses to a `32 CRF`, but this is configurable with
-the [`videoCompression`](/guides/references/configuration#Videos) property.
+file size. By default videos do not compress, but this is configurable with the
+[`videoCompression`](/guides/references/configuration#Videos) property.
```

Comment:
I think we should reword this more since starting with "After `cypress run` completes, Cypress compresses the video..." doesn't really make sense any more.

Suggestion: "After `cypress run` completes, videos are not compressed by default. However, compression can be enabled and configured with the [`videoCompression`](/guides/references/configuration#Videos) property in order to save on file size."

Reply:
OK I gave this a reword in f94affcc7